diff --git a/node_modules/@expo/xdl/.DS_Store b/node_modules/@expo/xdl/.DS_Store
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/@expo/xdl/build/Webpack.js b/node_modules/@expo/xdl/build/Webpack.js
index a8470e4..2955c64 100644
--- a/node_modules/@expo/xdl/build/Webpack.js
+++ b/node_modules/@expo/xdl/build/Webpack.js
@@ -201,7 +201,7 @@ const HOST = _getenv().default.string('WEB_HOST', '0.0.0.0');
 
 exports.HOST = HOST;
 
-const DEFAULT_PORT = _getenv().default.int('WEB_PORT', 19006);
+const DEFAULT_PORT = _getenv().default.int('WEB_PORT', 3000);
 
 exports.DEFAULT_PORT = DEFAULT_PORT;
 const WEBPACK_LOG_TAG = 'expo';
